docs/interceptors.md

```diff
@@ -74,7 +74,7 @@ OkHttpClient client = new OkHttpClient.Builder()
     .build();
 
 Request request = new Request.Builder()
-    .url("http://www.publicobject.com/helloworld.txt")
+    .url("https://www.publicobject.com/helloworld.txt")
```

Comment:
Sorry, I missed this in previous review. Given documentation below, I think this line should be reverted.